src/pivot-table/hooks/use-column-width/use-column-width-left.ts

```diff
@@ -31,10 +31,38 @@ export default function useColumnWidthLeft({ layoutService, tableRect, headersDa
 
   const calculateLeftGridWidthInfo = useCallback(
     (widthOverride?: number, overrideIndex?: number) => {
-      const maxMeasureCellWidth = qMeasureInfo.reduce((maxWidth, { qFallbackTitle, columnWidth }) => {
-        const fitToContentWidth = measureTextForMeasureValue(qFallbackTitle) + TOTAL_CELL_PADDING;
-        return Math.max(maxWidth, getColumnWidthValue(tableRect.width, columnWidth, fitToContentWidth));
-      }, 0);
+      const dimensionHeaderCellWidth = (lastRowHeader: HeaderCell) => {
+        const { label, isLocked } = lastRowHeader;
+        const lockedIconSize = isLocked ? LOCK_ICON_SIZE : 0;
+        return TOTAL_CELL_PADDING + measureTextForHeader(label) + MENU_ICON_SIZE + lockedIconSize;
+      };
+
+      const measureFitToContentWidth = (fitToContentWidth = 0) =>
```

Comment:
is this really the fitToContent width? you apply a value as argument, then call getColumnWidthValue. I see why you want to get it as that, but this would be the final width for a column, when you take all measures into account

Reply:
Yeah it's just that it's not necessarily a fitToContent value you get back from this function. If a measure has specified a width in e.g. pixel and that is the widest one, that is the value this function returns. so it is basically `maxMeasureColumnWidth` or something

Reply:
Got it. I'll change the name

Reply:
I've changed the name